src/basic/macro.h

```diff
@@ -222,6 +222,15 @@ static inline unsigned long ALIGN_POWER2(unsigned long u) {
                 UNIQ_T(A,aq) > UNIQ_T(B,bq) ? UNIQ_T(A,aq) - UNIQ_T(B,bq) : 0; \
         })
 
+#define CMP(a, b) __CMP(UNIQ, (a), UNIQ, (b))
+#define __CMP(aq, a, bq, b)                             \
+        __extension__ ({                                \
```

Comment:
we don't need __extension__ really, we don't compile with `-pedantic`... We use all sorts of gcc extensions, hence we don't need to mark their use in particular...

Reply:
So I was asking myself about it... But in the end decided to follow what MAX, MIN and the other friends do... Should we remove it from those as well?

Reply:
i figure, yes, please